src/cljs/athens/events.cljs

```diff
@@ -1164,8 +1164,27 @@
 
 (reg-event-fx
   :indent
-  (fn [_ [_ uid d-event]]
-    (indent uid d-event)))
+  (fn [_ [_ {:keys [uid d-key-down] :as args}]]
+    (js/console.debug ":indent" args)
+    (let [local?                    (not (client/open?))
+          block                     (common-db/get-block @db/dsdb
+                                                         (common-db/e-by-av @db/dsdb :block/uid uid))
```

Comment:
You could also use here `[:block/uid uid]` instead of calling `common-db/e-by-av`.